.npmignore

```diff
@@ -1,44 +1,23 @@
-*.scssc
-*.zip
-*.gzip
-*.tgz
-*.DS_Store
-
-*.sass-cache
-*COMPILE.css
-
-# Intellij project configuration files
-*.idea
-*.iml
-
-# External dependencies
-
-# Build output
-target
-
-# Mac OS X Finder
-.DS_Store
-
-# Closed source libraries
-closed-lib
-
-# Node, Bower dependencies
-node_modules
-bower_components
-
-Procfile
-
-# Protractor logs
-protractor/logs
-
-# npm-debug log
-npm-debug.log
-
-# Infra and tests
-.circleci
-.github
-e2e
-codecov.yml
-lighthouserc.yml
-*.Spec.js
-karma.conf.js
+# Ignore everything first (will not ignore special files like LICENSE.md,
+# README.md, and package.json)...
+/**/*
+
+# ...but include these folders...
+!/dist/**/*
+!/src/**/*
+!/platform/**/*
+
+# ...except for these files in the above folders.
+/src/**/*Spec.js
+/src/**/test/
+# TODO move test utils into test/ folders
+/src/utils/testing.js
+/platform/**/*Spec.js
+/platform/**/test/
```

Comment:
can we remove the platform directories? they're no longer in the codebase

Reply:
done